src/main/java/uk/gov/hmcts/reform/sendletter/data/LetterMapperFactory.java

```diff
@@ -33,7 +33,8 @@ public LetterStatus mapRow(ResultSet rs, int rowNumber) throws SQLException {
                 rs.getString("message_id"),
                 getDateTime(rs, "created_at"),
                 getDateTime(rs, "sent_to_print_at"),
-                getDateTime(rs, "printed_at")
+                getDateTime(rs, "printed_at"),
+                rs.getBoolean("is_failed")
```

Comment:
if SQL value is NULL, false is returned

Reply:
in migration script did not see the default value (or null). assume it is always boolean as per column type